omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,175 +51,280 @@ running::
-Ice 3.6.
+Local built server
+^^^^^^^^^^^^^^^^^^
 
-Prepare a place for your OMERO code to live, e.g.::
+1. Prepare a place for your OMERO code to live, e.g.::
 
-    $ mkdir -p ~/code/projects
-    $ cd ~/code/projects
+    $ mkdir -p ~/Projects/Omero/code
+    $ cd ~/Projects/Omero/code
 
-If you want the development version of OMERO.server, you can clone the source
-code from the project's GitHub account to build locally::
+2. Clone the source code from the project's GitHub account to build locally::
 
     $ git clone --recursive git://github.com/openmicroscopy/openmicroscopy
-    $ cd openmicroscopy && ./build.py
 
-.. note::
-    If you have a GitHub account and you plan to develop code for OMERO, you
-    should make a fork into your own account and then clone this fork to your
-    local development machine, e.g.
+3. Navigate terminal into the :file:`openmicroscopy` that was just created by performing
+   the previous step::
 
-    ::
+    $ cd openmicroscopy
 
-        $ git remote add  git://github.com/YOURNAMEHERE/openmicroscopy
-        $ cd openmicroscopy && ./build.py
+4. Execute the build script *(this will take a few minutes, depending on how fast your Mac is)* :: 
 
-.. seealso::
+    $ ./build.py
 
-    :doc:`/developers/installation`
+  .. seealso::
+   :doc:`/developers/installation`
         Developer documentation page on how to check out to source code
-
-    :doc:`/developers/build-system`
+   :doc:`/developers/build-system`
         Developer documentation page on how to build the OMERO.server
 
+5. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/openmicroscopy/dist`.
+   If it is not already open, open your :file:`.bash_profie`::
+
+    $ open -a TextEdit.app ~/.bash_profile
+
+   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+
+    export PATH=~/Projects/Omero/code/openmicroscopy/dist/bin:$PATH
+
+   Using the command-line terminal, reload your :file:`.bash_profile` using::
+
+    $ source ~/.bash_profile
+
+   To ensure OMERO is correctly linked into your OS X :envvar:`PATH`, type the following in terminal and ensure
+   you get a similar output::
+
+    $ which omero
+    /Projects/omero/code/openmicroscopy/dist/bin/omero
+
+6. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
+
+    $ source ~/Virtual/Omero/bin/activate
+
+7. Install Python dependencies using pip::
+
+    $ pip install -r ~/Projects/omero/code/openmicroscopy/dist/share/web/requirements-py27-all.txt
+
+
+OMERO configuration
+-------------------
+
+1. From a fresh command-line terminal, start the database server::
+
+    $ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start
+
+  .. note::
+   **(Optional)** To make life easier, you can add an ```alias``` to your :file:`.bash_profile`::
+    
+    # Start Virtualenv for OMERO
+    alias startVmOmero="source ~/Virtual/Omero/bin/activate"
```

Comment:
I am being offered to create this alias already for the second time. The first time it was https://github.com/openmicroscopy/ome-documentation/pull/1766/files#diff-38c0f35540194fd0f8261d5f85fae3fcR163